react/src/base/inputs/DatePicker.stories.js

```diff
@@ -1,132 +1,298 @@
+  name: 'Huge',
+  parameters: {
+    jest: ['SprkInputContainer', 'SprkDatePicker', 'SprkLabel'],
+  },
+};
+
+export const invalidHugeDatePicker = () => (
+  <SprkInputContainer variant="huge">
+    <SprkDatePicker
+      id="huge-datepicker-2"
+      variant="huge"
+      placeholder="01/01/2021"
+      isValid={false}
+      ariaDescribedBy="invalid-huge"
+    />
+    <SprkLabel htmlFor="huge-datepicker-2">Date</SprkLabel>
+    <SprkFieldError id="invalid-huge">
+      <SprkIcon
+        iconName="exclamation-filled"
+        additionalClasses="sprk-b-ErrorIcon"
+        aria-hidden="true"
+      />
+      <div className="sprk-b-ErrorText">There is an error on this field.</div>
+    </SprkFieldError>
+  </SprkInputContainer>
+);
+
+invalidHugeDatePicker.story = {
+  name: 'Huge Invalid',
+  parameters: {
+    jest: [
+      'SprkFieldError',
+      'SprkInputContainer',
+      'SprkDatePicker',
+      'SprkLabel',
+      'SprkIcon',
+    ],
+  },
+};
+
+export const disabledHugeDatePicker = () => (
+  <SprkInputContainer variant="huge">
+    <SprkDatePicker
+      id="huge-datepicker-3"
+      variant="huge"
+      placeholder="01/01/2021"
+      isDisabled
+    />
+    <SprkLabel htmlFor="huge-datepicker-3" isDisabled>
+      Date
+    </SprkLabel>
+  </SprkInputContainer>
+);
+
+disabledHugeDatePicker.story = {
+  name: 'Huge Disabled',
+  parameters: {
+    jest: ['SprkInputContainer', 'SprkDatePicker', 'SprkLabel'],
+  },
+};
+
+export const legacyDefaultStory = () => (
   <SprkDatePickerInput
     name="date"
-    placeholder="01/01/2019"
+    placeholder="01/01/2021"
     errorMessage="Incorrect date."
     label="Date"
   />
 );
 
-defaultStory.story = {
-  name: 'Default',
+legacyDefaultStory.story = {
+  name: 'Legacy (Deprecated)',
   parameters: {
     jest: ['SprkDatePickerInput'],
   },
 };
 
-export const invalidDatePicker = () => (
+export const legacyInvalidDatePicker = () => (
   <SprkDatePickerInput
     name="date"
-    placeholder="01/01/2019"
+    placeholder="01/01/2021"
     errorMessage="Incorrect date."
     label="Date"
     valid={false}
   />
 );
 
-invalidDatePicker.story = {
-  name: 'Invalid',
+legacyInvalidDatePicker.story = {
+  name: 'Legacy Invalid (Deprecated)',
   parameters: {
     jest: ['SprkDatePickerInput'],
   },
 };
 
-export const disabledDatePicker = () => (
+export const legacyDisabledDatePicker = () => (
   <SprkDatePickerInput
     name="date"
-    placeholder="01/01/2019"
+    placeholder="01/01/2021"
     errorMessage="Incorrect date."
     label="Date"
     disabled
   />
 );
 
-disabledDatePicker.story = {
-  name: 'Disabled',
+legacyDisabledDatePicker.story = {
+  name: 'Legacy Disabled (Deprecated)',
   parameters: {
     jest: ['SprkDatePickerInput'],
   },
 };
 
-export const hugeDatePicker = () => (
+export const legacyHugeDatePicker = () => (
   <SprkDatePickerInput
     type="hugeTextInput"
     name="date"
-    placeholder="01/01/2019"
+    placeholder="01/01/2021"
```

Comment:
Switching to MM/DD/YYYY like angular and react will solve the out of date issue with this too.